### ISSUE TITLE
Crash on attempt to browse folder from Uploads tab v.3.3.2 

### ISSUE THREAD

**Issue by 501NotImplemented** [nicotine-plus/nicotine-plus#2914]
Nicotine+ Version: 3.3.2
GTK Version: 4.12.5
Python Version: 3.11.8 (win32)

Windows 11 Pro
Version	23H2
OS build	22631.3235
Experience	Windows Feature Experience Pack 1000.22687.1000.0

```
Type: <class 'TypeError'>
Value: can only concatenate str (not "int") to str
Traceback:   File "D:/a/nicotine-plus/nicotine-plus/pynicotine/gtkgui/application.py", line 874, in _raise_exception
  File "C:/msys64/mingw64/lib/python3.11/threading.py", line 1045, in _bootstrap_inner
  File "C:/msys64/mingw64/lib/python3.11/threading.py", line 982, in run
  File "D:/a/nicotine-plus/nicotine-plus/pynicotine/userbrowse.py", line 99, in _parse_local_shares
  File "D:/a/nicotine-plus/nicotine-plus/pynicotine/slskmessages.py", line 3029, in parse_network_message
  File "D:/a/nicotine-plus/nicotine-plus/pynicotine/slskmessages.py", line 3064, in _parse_network_message
  File "D:/a/nicotine-plus/nicotine-plus/pynicotine/slskmessages.py", line 3046, in _parse_result_list
  File "D:/a/nicotine-plus/nicotine-plus/pynicotine/slskmessages.py", line 308, in unpack_string
```

### Describe the bug
Nicotine+ 3.3.2 crashes on an attempt to browse the folder from the Uploads tab

### Expected behavior
The app should continue to work. The folder should be browsed in the Browse Files tab.

### Steps to reproduce the bug
1. Open Uploads tab
2. Find active upload to user
3. Right click on the username
4. Click 'Browse folder'

### Additional context
Screenshots, logs, stacktraces or relevant information.
In fact I've misclicked 'Browse folder' from the context menu of user in Uploads, instead of browsing user files, I've attempted to browse my own files and it immediately crashed an app.

**Comment by mathiascode** [comment]
Probably related to https://github.com/nicotine-plus/nicotine-plus/issues/2907. Something seems to be wrong with your shares.

Can you send all your .dbn files to my email address (on my GitHub profile)? They are located in C:\Users\USERNAME\AppData\Roaming\nicotine.

**Comment by 501NotImplemented** [comment]
@mathiascode done. Please check your email.

**Comment by slook** [comment]
Closing as duplicate of resolved issue #2907, at least the problem that leads to the original crash will be avoided since https://github.com/nicotine-plus/nicotine-plus/commit/e8b8a910582b1fd3b99e514b3c4e561c399e7bb1.
